mypy/nodes.py

```diff
@@ -155,9 +155,6 @@ def accept(self, visitor: StatementVisitor[T]) -> T:
 
 class Expression(Node):
     """An expression node."""
-    literal = LITERAL_NO
-    literal_hash = None  # type: Key
```

Comment:
This is an exception for historical reasons. Before PEP 526 there was no way do define an instance variable without a default value ``x: int``. Therefore it is allowed to write this even with ``--strict-optional`` (the same code should fail at module and function scopes).